### PR TITLE
Update Test Suite Level Visibility support version for Java

### PR DESCRIPTION
**Pull request by nikita-tkachenko-datadog**
### What does this PR do?

Updates CI Visibility Tests product docs. 
Compatibility section for Test Suite Level Visibility feature now mentions the version of DD Java tracer in which support for the feature was added.
Also, a definition of Test Module is given for Java tracer.

### Motivation

A new version of DD Java Tracer was released, that supports Test Suite Level Visibility Feature

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/tests/_index.md [hunk 1]
```diff
@@ -104,6 +104,7 @@ The definition of module changes slightly per language:
 * In .NET a test module groups every test that is run under the same [unit test project][8].
 * In Swift a test module groups every test that is run for a given bundle.
 * In JavaScript there are no test modules.
+* In Java a test module groups every test that is run by the same Maven Surefire/Failsafe or Gradle Test task execution
```
Comment (drichards-87) [inline content/en/continuous_integration/tests/_index.md:107]:
```suggestion
* In Java a test module groups every test that is run by the same Maven Surefire/Failsafe or Gradle Test task execution.
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left you one minor piece of feedback. Also, let me know if this ready to merge after you update the PR. Thanks!

**Comment by nikita-tkachenko-datadog** [comment]
> Left you one minor piece of feedback. Also, let me know if this ready to merge after you update the PR. Thanks!

Thank you for the review! The changes are ready for merge from my perspective.

**Comment by drichards-87** [comment]
@nikita-tkachenko-datadog Sounds good. It looks like we have a merge conflict in the PR. Would you be able to take a look at it? I can also take a look as well, but it might make more sense for you to do so since you have more context than I do.

**Comment by nikita-tkachenko-datadog** [comment]
> @nikita-tkachenko-datadog Sounds good. It looks like we have a merge conflict in the PR. Would you be able to take a look at it? I can also take a look as well, but it might make more sense for you to do so since you have more context than I do.

Done, conflict resolved

**Review by drichards-87 — APPROVED** [review]

**Pull request merged**